tiles/src/main/java/com/protomaps/basemap/layers/Water.java

```diff
@@ -43,9 +43,18 @@ public void processFeature(SourceFeature sf, FeatureCollector features) {
         .setAttr("leisure", sf.getString("leisure"))
         .setAttr("water", sf.getString("water"))
         .setAttr("waterway", sf.getString("waterway"))
+        // Add less common attributes only at higher zooms
```

Comment:
These props mostly apply to water lines, but there are exceptions for polygons, too, so are included here for parity.